lib/index.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
```

Comment:
This file needs to be ignored.

Reply:
Please clarify/elaborate. I am unsure what you mean. Change tsconfig.json so that no `use strict` is emited?

Reply:
Oh... You probaly mean that *.js gets added to .gitignore, correct?